src/Raven.Pal/.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "type": "shell",
+            "command": "/home/ayende/zig/zig-linux-x86_64-0.14.0-dev.2316+68b3f5086/zig",
+            "args": [
+                "cc",
+                "-Wall",
+                "-O0",
+                "-g",
+                "-fPIC",
+                "-Iinc",
+                "-target",
+                "x86_64-linux-gnu",
+                "../../libs/liburing/liburing-2.8.1-x64.a",
```

Comment:
can we write a comment 'somewhere' from where both of those were taken?

Reply:
Note that this is basically just debug code, but sure.

Reply:
Actually, no way to comment here 🤦 , JSON

Same as the VS project, that is for building / working etc.

Reply:
I more meant a link to repo from where this lib is coming from, couldnt find a better place to write this comment, but Arek is asking for same thing: https://github.com/ayende/ravendb/pull/4696#discussion_r1910331614

Reply:
Done